api/reqs/tests/views_policies_tests.py

```diff
@@ -180,3 +180,15 @@ def test_pk_id():
     mommy.make(Policy, pk=pk_id)
     response = client.get(path + '.json').json()
     assert response['id'] == pk_id
+
+
+@pytest.mark.django_db
+def test_slug():
+    client = APIClient()
+    slug = "hello-there"
+    path = "/policies/{0}".format(slug)
+    response = client.get(path)
+    assert response.status_code == 301
```

Comment:
Should this be a 404? Maybe it's not following a redirect to `/policies/hello-there/`?

Reply:
Ummm to be honest I was just copy-pasting the previous test, which tested to make sure that retrieving policies by OMB policy ids worked. I would've expected it to be a 404 myself... maybe the path should end with a `/` and it should verify that it 404's?